src/components/input.js

```diff
@@ -1,14 +1,29 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 export default class Input extends React.Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
 
+  handleChange = (e) => {
+    const { display } = this.props;
+    e.target.value = display;
+  }
+
   render() {
+    const { display } = this.props;
     return (
-      <input type="text" className="calc-inp" value="0" />
+      <input type="text" className="calc-inp" value={display} onChange={this.handleChange} />
```

Comment:
- [ ] When a number is divided by 0, your app throws an error on the browser. Kindly incorporate error handling conditions. 
![image](https://user-images.githubusercontent.com/64894927/129421711-abb51421-6995-4593-9bfb-817d90cb3ea4.png)